twitter/View Controllers/TimelineViewController.m

```diff
@@ -58,9 +69,11 @@ - (void)didReceiveMemoryWarning {
     [super didReceiveMemoryWarning];
 }
 
-- (void)didTweet:(Tweet *)tweet {
-    [self.tweetModels insertObject:[[TweetCellModel alloc] initWithTweet:tweet] atIndex:0];
-    [self.homeTweetTableView reloadData];
+- (void)postTweet:(Tweet *)tweet {
+    if (tweet.repliedToTweet == nil || [tweet.repliedToTweet isEqual:[NSNull null]]) {
```

Comment:
My thought reading this is, what is the abstraction behind this? What does it mean if a tweet has no replied to tweet?

If I were to write this I would probably have a field on Tweet itself to tell me if it's a reply. Then here you can just ask it.